app/containers/NewApp/index.js

```diff
@@ -35,16 +35,10 @@ export default function App() {
         titleTemplate="%s - Charity activity"
         defaultTitle="Charity activity"
       >
-        <meta name="description" content="A React.js Boilerplate application" />
+        <meta name="description" content="Charity activity" />
       </Helmet>
       <Menu />
-      <Title />
-      <Header />
-      <Switch>
-        <Route exact path="/" component={HomePage} />
-        <Route path="/events" component={() => <span>events</span>} />
-        <Route path="" component={NotFoundPage} />
-      </Switch>
+      <Hero />
```

Comment:
I'm wondering is it displayed correctly ? I mean you've removed all of the routing part - that's why I'm asking. If you are not sure, don't worry let me know then I'm gonna check it.

Reply:
Right... bad decision.

Reply:
Are we ready to approve & merge it ?